modules/apps/29-fee/keeper/escrow.go

```diff
@@ -137,14 +138,12 @@ func (k Keeper) DistributePacketFeesOnTimeout(ctx sdk.Context, timeoutRelayer sd
 
 // distributePacketFeeOnTimeout pays the timeout fee to the timeout relayer and refunds the acknowledgement & receive fee.
 func (k Keeper) distributePacketFeeOnTimeout(ctx sdk.Context, refundAddr, timeoutRelayer sdk.AccAddress, packetFee types.PacketFee) {
-	// refund receive fee for unused forward relaying
-	k.distributeFee(ctx, refundAddr, refundAddr, packetFee.Fee.RecvFee)
-
-	// refund ack fee for unused reverse relaying
-	k.distributeFee(ctx, refundAddr, refundAddr, packetFee.Fee.AckFee)
-
 	// distribute fee for timeout relaying
 	k.distributeFee(ctx, timeoutRelayer, refundAddr, packetFee.Fee.TimeoutFee)
+
+	// refund unused fees
+	refundCoins := packetFee.Fee.Total().Sub(packetFee.Fee.TimeoutFee...)
+	k.distributeFee(ctx, refundAddr, refundAddr, refundCoins)
```

Comment:
hmm, this isn't backwards compatible? If I escrow with the old logic and distribute with the new logic, I'll be short changed?

Reply:
Oh we can just do a migration. Refund `oldTotal() - newTotal()`. That sounds the easiest to me, as opposed to handling a legacy case

Reply:
I thought about and mentioned on the issue but wanted to leave it up to broader team discussion. I'm actually ok with these fees being short charged and effectively burned.

Reply:
added migration